libs/blocks/card/cardUtils.js

```diff
@@ -36,10 +36,17 @@ export const addFooter = (links, container, merch) => {
   });
 };
 
-export const addWrapper = (el, section, cardType) => {
+export const addWrapper = (el, cardType) => {
+  const fragment = el.closest('.fragment');
+  if (fragment) {
+    const fragmentSection = el.closest('.section');
```

Comment:
This is called `fragmentSection`, but you're looking for the closest `.section` ancestor of the `el` element. Should `el` be switched to `fragment` or should the variable name be updated to `elSection` or whatever fits best? In the unit tests, it seems that `fragment` is used in this scenario, not `el`.

This might also be helpful in case the structure is sometimes `<fragment><section>` and other times `<section><fragment>` or other combinations of these elements.

Reply:
the structure is like this (card being the element):
```
<section>
   <fragment>
      <section>
         <card>
      <section>
   <fragment>
<section>
```

I agree 'elSection' is more clear, will rename, thx